src/deepsparse/debug_analysis.py

```diff
@@ -307,10 +349,31 @@ def main():
     orig_model_path = args.model_path
     model_path = model_to_path(args.model_path)
 
-    print("Analyzing model: {}".format(orig_model_path))
+    print(f"Analyzing model: {orig_model_path}")
 
     batch_size = args.batch_size
 
+    if has_model_kv_cache(model_path):
+        if batch_size != 1:
+            raise ValueError(
+                "Unable to run models with KV cache support "
+                "for batch size different than one."
+                "Please set batch size to 1 and try again"
```

Comment:
Should we use `--use-kvcache` here instead of querying `has_model_kv_cache`?

Reply:
`--use-kvcache` just enables the internal kv cache and we want to be able to measure the difference. Probably `--use-kvcache` should be renamed